frontend/src/features/public-form/components/FormIssueFeedback/FormIssueFeedbackContainer.tsx

```diff
@@ -0,0 +1,11 @@
+import { Box } from '@chakra-ui/react'
+
+import { FormIssueFeedbackButton } from './FormIssueFeedbackButton'
+
+export const FormIssueFeedbackContainer = (): JSX.Element | null => {
```

Comment:
```suggestion
export const FormIssueFeedbackContainer = (): JSX.Element => {
```
`null` type not required as the return type is constant here